### DISCUSSION TITLE
What??? Why can't I create my special repository for my Github profile?

### DISCUSSION THREAD

**Discussion by igui-git** [community/community#37786] [Other Feature Feedback, Questions, & Ideas]
**
![image](https://user-images.githubusercontent.com/61317250/198882264-065043c3-58fb-45ab-98d8-a789c648c061.png)
**

**Comment by RubenGamarrarodriguez-tomtom** [comment]
Try changing the name, for example to [gsrib](https://github.com/gsrib/gsrib)

Here is an example of what you can put in the profile
```html

<div align="center">

<img src="https://user-images.githubusercontent.com/61317250/193479171-c730acc2-defc-477c-82ff-b2400d073534.png" width="100%" height="90px" />

  # 👋🏻 Olá, meu nome é Guilherme Ribeiro<br>e também conhecido como <b>GR</b>!
  <code align=center>› Desenvolvedor à 3 anos. Vivo à 17 anos</code>
<br>
  <a href="https://sourceb.in/2AYgihfavN" alt="GR"><img src="https://skillicons.dev/icons?i=coffeescript,js,typescript,nodejs,firebase,mongo,discord,aws,html,css,sass,tailwind&theme=dark&perline=6"></a>
<br>
  <details>
    <summary><underline><b>› ・› Github Status..・</b></underline></summary>
    <table><tr><td style="padding: 0; width=50%">
      <img src="https://github-readme-stats.vercel.app/api/?username=gsrib&show_icons=true&title_color=1c6cbf&text_color=246af9&bg_color=00000000&hide_border=true&icon_color=1c6cbf&hide_title=true&count_private=true" /></td>
      <td style="padding: 0; width=50%"><img src="https://github-readme-stats.vercel.app/api/top-langs/?username=gsrib&show_icons=true&title_color=1c6cbf&text_color=246af9&bg_color=00000000&hide_border=true&icon_color=00000000&count_private=true" /></td></tr></table>
  </details> 
  <br>
  <a href="https://discord.com/users/424931675009712128" alt="GR"><img src="https://lanyard.cnrad.dev/api/424931675009712128" /></a>
  
  <code align=center>› Contate-me para mais para informações</code>
  
  <code align=center>› Email, Discord e Twitter anexado ao perfil</code>

</div>
```

**Reply by igui-git** [reply]
But I would not like to change users. I want to use username `guihrib` and my special profile with repository name `guihrib` too. **But why can't I change?** I see no plausible justification

**Comment by mirzohidsobirjonov** [comment]
There is repository like guihrib. That is why You cannot create. Please change the name of repository and try again.

**Reply by igui-git** [reply]
![image](https://user-images.githubusercontent.com/61317250/198892814-b2fbd46a-f622-4e7a-a43d-b74abe8801a0.png)

I can't use the username `guihrib` and have a repository named `guihrib`, that's the fact. If I have the different username and create the `guihrib` repository it works (And that's how I'm currently doing it). But if I tried to do what I said at the beginning of this answer, it doesn't work. and why this happens? Was I punished? Whereby? It is a mistake? Can you solve? **I just want to use the username guihrib and have my special repository named guihrib, not something other than that.**

**Comment by jfullstackdev** [comment]
what I can see is that, you already created same repository and deleted it and wants to recreate it, that's why it's retired

https://github.blog/2018-04-18-new-tools-for-open-source-maintainers/#popular-repository-namespace-retirement

GitHub Support might help but there is no assurance

https://support.github.com/contact/account
